src/devices/gridstream.c

```diff
@@ -0,0 +1,258 @@
+/** @file
+    Decoder for Gridstream RF devices produced by Landis & Gyr.
+
+    Copyright (C) 2016 Benjamin Larsson
```

Comment:
I doubt that I wrote this decoder in 2016. Change so that attribution is correct.

Reply:
Addressed in https://github.com/merbanan/rtl_433/pull/2616/commits/7edd93a5aead3c669f4bd8e5dbf2d6d32e9ede96